src/webgl.js

```diff
@@ -1591,13 +1590,18 @@ export function main(player, canvas, options) {
       frameCount = 0;
       elapsedTime -= 1000;
 
+      // frameDelay minimum of 1 avoids the frameDelay getting stuck at zero in the update expression below
+      if (frameDelay < 1) {
+        frameDelay = 1
+      }
+      // set frameDelay based on the current value and how close we are to targetFrameRate
+      frameDelay = frameDelay * fps / targetFrameRate;
+
```

Comment:
Note, I didn't check this math... assuming you did :P

Reply:
There might be a better formula, this one has the benefit of being simple so it's fast and easy to understand. And it was the first one I tried and seemed to work pretty well. 😄